react/features/toolbox/components/web/Toolbox.js

```diff
@@ -975,7 +980,7 @@ class Toolbox extends Component<Props, State> {
             <RecordButton
                 key = 'record'
                 showLabel = { true } />,
-            this._shouldShowButton('sharedvideo')
+            isModerator && !this._shouldShowButton('sharedvideo')
```

Comment:
@DeeJayElly probably you got me wrong about what `we don’t need more`. We don’t need to remove it from the `interface_config` and now also we don’t need a bang(!) before the `this._shouldShowButton `
Otherwise we will get the same issue as it was described at the first review.
https://github.com/aeternity/jitsi-meet/pull/17#pullrequestreview-422750358

Reply:
Fixed